packages/@react-aria/numberfield/src/useNumberField.ts

```diff
@@ -72,7 +73,8 @@ export function useNumberField(props: AriaNumberFieldProps, state: NumberFieldSt
     autoFocus,
     validationState,
     label,
-    formatOptions
+    formatOptions,
+    disableScroll
```

Comment:
Do we really need an option for this or is disabling when focused enough?

Reply:
you mean disabling when not focused?
it came from your comment https://github.com/adobe/react-spectrum/issues/1602#issuecomment-781339005
so I think you're saying you now think that completely disabling it not a needed feature